packages/react-components/components/Link.tsx

```diff
@@ -7,7 +7,7 @@ type Props = TouchableOpacityProps & {
   testID?: string
 }
 
-export default class Link extends React.Component<Props> {
+export default class Link extends React.PureComponent<Props> {
```

Comment:
nit: is it really needed?

Reply:
Not needed but IIUC there is a benefit to having these small components be Pure to prevent re-renders. Any reason they _shouldn't_ be Pure?

Reply:
Either way I think it probably doesn't matter here.
The rule I try follow is to use `PureComponent` when I've identified and measured a real bottleneck as explained by Dan Abramov https://twitter.com/dan_abramov/status/759383530120110080

Reply:
I'd love to understand this better because it doesn't really align with my intuition, which is that shallow equality checks should be trivial.
IIUC it comes down to this bit that Abramov points out:
> Basically compare + render anyway is slower than just render anyway
Which makes some sense, but I'd still expect the compare to be trivial.